state/protocol/badger/snapshot.go

```diff
@@ -378,6 +379,24 @@ func (s *Snapshot) Params() protocol.GlobalParams {
 	return s.state.Params()
 }
 
+func (s *Snapshot) VersionBeacon() (*flow.SealedVersionBeacon, error) {
+	head, err := s.state.headers.ByBlockID(s.blockID)
+	if err != nil {
+		return nil, err
+	}
+
+	versionBeacon, err := s.state.versionBeacons.Highest(head.Height)
+
+	if err != nil {
+		if errors.Is(err, storage.ErrNotFound) {
+			return nil, state.ErrNoVersionBeacon
```

Comment:
this looks weird.  maybe just have the Highest function directly return ErrNoVersionBeacon instead of ErrNotFound

Reply:
I added a commit getting rid of the `ErrNoVersionBeacon` entirely. It simplifies quite a bit